bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/fs/watch/JSScriptFileWatcher.java

```diff
@@ -68,9 +66,4 @@ public Optional<String> getScriptType() {
             }
         });
     }
-
-    @Override
-    protected boolean watchSubDirectories() {
```

Comment:
This change means that sub directories are watched now.
I have no strong preference regarding this behaviour, so I'm fine with it.

Does dependency and script file tracking still work as expected?

Reply:
This is preserved for you. It's just a `false` argument when starting the watch service. I double checked to make sure the Jruby watchers were still watching subdirectories.